### PR TITLE
become just the block-service

### PR DESCRIPTION
**Pull request by daviddias**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
-  "name": "ipfs-blocks",
+  "name": "ipfs-block-service",
   "version": "0.2.3",
```
Comment (dignifiedquire) [inline package.json:3]: maybe change the version?

File: src/index.js [hunk 2]
```diff
@@ -1,5 +1,4 @@
 'use strict'
 
-exports.BlockService = require('./block-service.js')
-exports.Block = require('./block.js')
+exports = module.exports = require('./block-service.js')
 exports.util = require('./util.js')
```
Comment (dignifiedquire) [inline src/index.js:4]: util can be removed that's not used here

File: package.json (hunk 1 above)
Comment (daviddias) [inline package.json:3]: happens during release process

File: src/index.js (hunk 2 above)
Comment (daviddias) [inline src/index.js:4]: ok, will do that in master then
Thank you :)

**Pull request merged**